sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -382,7 +391,7 @@ def send_to_connection(  # pylint: disable=inconsistent-return-statements
         _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
         _params = kwargs.pop("params", {}) or {}
 
-        content_type = kwargs.pop("content_type", _headers.pop("Content-Type", "application/json"))  # type: str
+        content_type = _headers.pop("Content-Type", "application/json") if content_type is None else content_type
```

Comment:
I think you could only add method signature and keep original logic about content_type since original logic permit users set content_type with `headers` which is common behavior for all python SDK.